octotiger/unitiger/hydro_impl/flux.hpp

```diff
@@ -9,6 +9,8 @@
 #include "octotiger/unitiger/physics.hpp"
 #include "octotiger/unitiger/physics_impl.hpp"
 
+//#define FACE_ONLY_HYDRO
+
```

Comment:
If we want it to be really a user option, we should convert it to a runtime parameter or at least a cmake variable.